docs/hub/oauth.md

```diff
@@ -0,0 +1,45 @@
+# Sign in with Hugging Face
+
+Besides [spaces](https://huggingface.co/docs/hub/spaces-oauth), it's also possible to create an OAuth / OpenID connect app for your website. This will allow users to sign-in to your website using their HF account.
+
+It's also possible to have the additional `email` scope.
```

Comment:
I would maybe add a small explanation that we don't provide access to write repos for example, which will be one of the most common requested use cases